### ISSUE TITLE
Bodhi crashes while composing a modular repo if it has a mailing list defined

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#2396]
Bodhi was [recently configured to have a mailing list configured for ```fedora_modular```](https://pagure.io/fedora-infrastructure/issue/6872). The next time a modular repo was composed, it failed with this traceback:

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/bodhi/server/consumers/masher.py", line 337, in run
    self.work()
  File "/usr/lib/python2.7/site-packages/bodhi/server/consumers/masher.py", line 422, in work
    self.send_stable_announcements()
  File "/usr/lib/python2.7/site-packages/bodhi/server/consumers/masher.py", line 70, in wrapper
    retval = method(self, *args, **kwargs)
  File "/usr/lib/python2.7/site-packages/bodhi/server/consumers/masher.py", line 732, in send_stable_announcements
    update.send_update_notice()
  File "/usr/lib/python2.7/site-packages/bodhi/server/models.py", line 2777, in send_update_notice
    for subject, body in mail.get_template(self, templatetype):
  File "/usr/lib/python2.7/site-packages/bodhi/server/mail.py", line 354, in get_template
    use_template = globals()[use_template]
KeyError: u'fedora_modular_errata_template'
```

This is because Bodhi's e-mail system hard codes releases into the Python code rather than using settings, and the modular repo isn't hard coded :/

**Comment by bowlofeggs** [comment]
This one is tricky. The problem is that some release data in *hardcoded* in ```mail.py```, which it absolutely should not be. We could solve this a few ways:

0. We could move these templates to production.ini, but then we exacerbate the problem of variable variable names that already exists there.
1. We could add a database column on the Release model that contains a filesystem path to a template to use for the release.
2. We could store the template itself in the database, and give ```bodhi releases``` the ability to edit it.
3. We could make a default template that gets used when no more specific template is found, so that no matter what it at least has a simple template to use.

**Comment by ghost** [comment]
@cverna: Is there an email template for ```fedora_modular```?

**Comment by cverna** [comment]
@vismay-golwala you can just copy the `fedora_errata_template` and replace  `Fedora%(testing)s Update Notification` by `Fedora Modular%(testing)s Update Notification`

**Comment by cverna** [comment]
@vismay-golwala and I think that for now the default_template can be `fedora_errata_template`

**Comment by bowlofeggs** [comment]
A fix for this issue is planned to be included in the upcoming 3.9.0 release: #2477.

**Comment by bowlofeggs** [comment]
The patch for this issue was deployed to Fedora's production Bodhi instance yesterday.
